cmd/install/assets/assets.go

```diff
@@ -13,6 +13,7 @@ import (
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+//go:embed prereqs-azure/*
```

Comment:
Where those CRDs come from? I mean, for self-maintenance would we need to add a command to auto-generation/update of those CRDs in the Makefile, like we do with CAPI ones?

Reply:
That's a good idea! I wasn't aware we do that. I'll check it out.